pages/common/vifm.md

```diff
@@ -10,3 +10,23 @@
 - Open specified directories on left or right plane:
 
 `vifm {{path/to/directory_1}} {{path/to/directory_2}}`
+
+- Copy, paste, move and delete:
+
+`yy, p, P and dd`
```

Comment:
this example sounds strange too me. also I think should be an interactive command to display help. that example should be in this page.